packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -514,27 +523,48 @@ function getRank(
     const decorated = 'decorators' in node && node.decorators!.length > 0;
     if (
       decorated &&
-      (type === 'field' ||
+      (type === 'readonly-field' ||
+        type === 'field' ||
         type === 'method' ||
         type === 'get' ||
         type === 'set')
     ) {
       memberGroups.push(`${accessibility}-decorated-${type}`);
       memberGroups.push(`decorated-${type}`);
+
+      if (type === 'readonly-field') {
+        // if its readonly-field, push also its superset type
```

Comment:
I think these are self-evident from the code. Is there a particular reason you wanted them?

```suggestion
```

(If we do keep them in, nit: it's `it's` not `its`. They're possessive, not plural.)

Reply:
Actually the comment is added there to draw attention to the implementation, since I'm not really sure if it's the right way to do it. 

I'm glad the code already speak for itself, I will remove the comments in the update.